dpnp/dpnp_iface_sorting.py

```diff
@@ -107,16 +113,14 @@ def argsort(a, axis=-1, kind=None, order=None):
 
     Notes
     -----
-    For zero-dimensional arrays, if `axis=None`, output is a one-dimensional
+    For zero-dimensional arrays, if ``axis=None``, output is a one-dimensional
     array with a single zero element. Otherwise, an ``AxisError`` is raised.
 
     Limitations
     -----------
-    Parameters `order` is only supported with its default value.
-    Parameters `kind` can only be ``None`` or ``"stable"`` which
-    are equivalent.
-    Otherwise ``NotImplementedError`` exception will be raised.
-
+    Parameters `order` and `stable` are only supported with their default
+    values. Parameter `kind` can only be ``None`` or ``"stable"`` which are
+    equivalent. Otherwise ``NotImplementedError`` exception will be raised.
```

Comment:
An empty line is missing
```suggestion
    equivalent. Otherwise ``NotImplementedError`` exception will be raised.

```